app/views/liquids/_katex.html.erb

```diff
@@ -7,7 +7,7 @@
   <%= parsed_content %>
 </span>
 <% else %>
-<div class="katex-element">
+<div class="katex-element" style="overflow-x: auto;">
```

Comment:
Just a question on this: Is this inline for a specific purposes instead of it going into a stylesheet?

Reply:
Not really. We are doing it for several other Liquid tags too, and I sometimes find it a bit easier to have it right there when looking at the tag, but I can move it if there's a strong preference.

Reply:
I don't think we properly established it yet but we should avoid adding inline style to liquid tags because the content is hardcoded as `body_html` which limits future changes.  It's best to apply this to `KatexTag.scss`, please see my temp commit here https://github.com/maestromac/dev.to/commit/c14300bed5ac15c8fffc72301a6c39572a788371

Reply:
Thanks @maestromac!